manifests/init.pp

```diff
@@ -425,30 +425,34 @@
     }
     if $facts['os']['family'] == 'RedHat' {
       python::virtualenv { 'patroni':
-        version    => $python_venv_version,
-        venv_dir   => $install_dir,
-        virtualenv => 'virtualenv-3',
-        systempkgs => true,
-        distribute => false,
-        require    => Exec['patroni-mkdir-install_dir'],
+        version     => $python_venv_version,
+        venv_dir    => $install_dir,
+        virtualenv  => 'virtualenv-3',
+        systempkgs  => true,
+        distribute  => false,
+        environment => ["PIP_PREFIX=${install_dir}"],
```

Comment:
spec tests should reflect the new environment attribute

Reply:
That's non-trivial because the current spec tests don't test any resources, we just test that the catalog compiles.  Should I add full spec tests or just leave as simple compile test?